data/strings/types_strings.txt

```diff
@@ -17108,400 +16767,59 @@
-    pt-BR = Entrada de metrô
-    ro = Intrare în metro
-    ru = Вход в метро
-    sk = Vstup do metra
-    sv = Tunnelbana entré
-    sw = Lango la kuingilia stesheni ya reli
-    th = ทางเข้ารถไฟใต้ดิน
-    tr = Metro Girişi
-    uk = Вхід до метро
-    vi = Lỗi vào Metro
-    zh-Hans = 地铁入口
-    zh-Hant = 地鐵入口
-
-  [type.railway.subway_entrance.barcelona]
-    en = Subway
-    ar = محطة مترو
-    cs = Metro
-    da = Metro
-    de = U-Bahn-Eingang
-    el = Μετρό
-    es = Metro
-    eu = Metroa
-    fa = حمل و نقل
-    fi = Metro
-    fr = Métro
-    hu = Földalatti
-    id = Stasiun bawah tanah
-    it = Metrò
-    ja = 地下鉄
-    ko = 지하철
-    mr = भुयारी मार्ग
-    nb = T-bane
-    nl = Metro
-    pl = Metro
-    pt = Entrada de estação de metrolitano
-    pt-BR = Entrada de estação do metrô
-    ro = Metrou
-    ru = Метро
-    sk = Metro
-    sv = Tunnelbana
-    th = รถไฟ
-    tr = Metro
-    uk = Метро
-    vi = Ngầm
-    zh-Hans = 地铁入口
-    zh-Hant = 地鐵入口
-
-  [type.railway.subway_entrance.berlin]
-    en = Subway
-    ar = محطة مترو
-    cs = Metro
-    da = Metro
-    de = U-Bahn-Eingang
-    el = Μετρό
-    es = Metro
-    eu = Metroa
-    fa = حمل و نقل
-    fi = Metro
-    fr = Métro
-    hu = Földalatti
-    id = Stasiun bawah tanah
-    it = Metrò
-    ja = 地下鉄
-    ko = 지하철
-    mr = भुयारी मार्ग
-    nb = T-bane
-    nl = Metro
-    pl = Metro
-    pt = Entrada de estação de metrolitano
-    pt-BR = Entrada de estação do metrô
-    ro = Metrou
-    ru = Метро
-    sk = Metro
-    sv = Tunnelbana
-    th = รถไฟ
-    tr = Metro
-    uk = Метро
-    vi = Ngầm
-    zh-Hans = 地铁入口
-    zh-Hant = 地鐵入口
-
-  [type.railway.subway_entrance.kiev]
-    en = Subway
-    ar = محطة مترو
-    cs = Metro
-    da = Metro
-    de = U-Bahn-Eingang
-    el = Μετρό
-    es = Metro
-    eu = Metroa
-    fa = حمل و نقل
-    fi = Metro
-    fr = Métro
-    hu = Földalatti
-    id = Stasiun bawah tanah
-    it = Metrò
-    ja = 地下鉄
-    ko = 지하철
-    mr = भुयारी मार्ग
-    nb = T-bane
-    nl = Metro
-    pl = Metro
-    pt = Entrada de estação de metrolitano
-    pt-BR = Entrada de estação do metrô
-    ro = Metrou
-    ru = Метро
-    sk = Metro
-    sv = Tunnelbana
-    th = รถไฟ
-    tr = Metro
-    uk = Метро
-    vi = Ngầm
+    it = Stazione metropolitana
+    ja = 地下鉄入り口
+    ko = 지하철 입구
+    mr = भुयारी प्रवेशद्वार
+    nb = T-bane inngang
+    nl = Metro ingang
+    pl = Wejście do metra
+    pt = Entrada de metropolitano
+    pt-BR = Entrada de metrô
+    ro = Intrare în metro
+    ru = Вход в метро
+    sk = Vstup do metra
+    sv = Tunnelbana entré
+    sw = Lango la kuingilia stesheni ya reli
+    th = ทางเข้ารถไฟใต้ดิน
+    tr = Metro Girişi
+    uk = Вхід до метро
+    vi = Lỗi vào Metro
     zh-Hans = 地铁入口
     zh-Hant = 地鐵入口
 
+  [type.railway.subway_entrance.barcelona]
+    ref = type.railway.subway_entrance
+
+  [type.railway.subway_entrance.berlin]
+    ref = type.railway.subway_entrance
+
+  [type.railway.subway_entrance.kiev]
```

Comment:
@vng Does our code support the parent type translation if the full type translation is missing? If yes, then we don't need even these refs, right? E.g. `type.railway.subway_entrance` will cover all subtupes.

Reply:
I have seen a warning about missing type.railway.subway_entrance.warszawa translation before I added it so it isn't supported or there is a bug